clients/cobol-lsp-vscode-extension/src/services/util/FSUtils.ts

```diff
@@ -86,6 +86,12 @@ function globSearch(
     copybookName +
     ext;
   pattern = pattern + suffix;
+
+  // Special handling of //./ UNC path as glob module doesn't support this. refer: https://github.com/isaacs/node-glob
+  if (pattern.startsWith("//./") && process.platform === "win32") {
+    pattern = pattern.replace("//./", "//?/");
```

Comment:
this makes sure that a java process started with UNC path like \\.\c:\Users doesn't fail.
Can't find anything about this in doc